dev-workflow-ui/src/ch/ivyteam/workflowui/statistics/StatisticsTimeResolver.java

```diff
@@ -32,6 +32,78 @@ public static TimeResolution getResolutionForDuration(String timeDuration) {
     };
   }
 
+  public static TimeResolution getResolutionForDurationAndType(String timeDuration, String resolutionType) {
+    return switch (resolutionType) {
+      case "hour" -> new TimeResolution("hour", calculateHourDataPoints(timeDuration), "HH:00");
+      case "hour6" -> new TimeResolution("hour6", calculate6HourDataPoints(timeDuration), "MM-dd HH:00");
```

Comment:
maybe this different resolution types etc should be a constant. also the durations.
```java
interface Resolution {
  String HOUR6 = "hour6";
  ...
}
```